app/src/main/res/layout/fragment_fever_temperature_spot.xml

```diff
@@ -0,0 +1,174 @@
+                android:onClick="@{() -> vm.onClickMouth()}">
+
+                <TextView
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginLeft="18dp"
+                    android:layout_marginRight="18dp"
+                    android:gravity="center"
+                    android:letterSpacing="0.01"
+                    android:lineSpacingExtra="1sp"
+                    android:text="@string/symptom_report_fever_temperature_spot_mouth"
+                    android:textColor="@color/button_text_color"
+                    android:textSize="24sp"
+                    android:textStyle="bold"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent"
+                    app:layout_constraintBottom_toBottomOf="parent"/>
+            </androidx.constraintlayout.widget.ConstraintLayout>
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:layout_width="126dp"
+                android:layout_height="126dp"
+                android:background="@drawable/circle_button_background"
+                android:onClick="@{() -> vm.onClickEar()}">
+
+                <TextView
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginLeft="18dp"
+                    android:layout_marginRight="18dp"
+                    android:gravity="center"
+                    android:letterSpacing="0.01"
+                    android:lineSpacingExtra="1sp"
+                    android:text="@string/symptom_report_fever_temperature_spot_ear"
+                    android:textColor="@color/button_text_color"
+                    android:textSize="24sp"
+                    android:textStyle="bold"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent"
+                    app:layout_constraintBottom_toBottomOf="parent"/>
+            </androidx.constraintlayout.widget.ConstraintLayout>
+        </LinearLayout>
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="horizontal"
+                android:layout_marginTop="38dp"
+                android:gravity="center">
+
+                <androidx.constraintlayout.widget.ConstraintLayout
+                    android:layout_width="126dp"
+                    android:layout_height="126dp"
+                    android:layout_marginEnd="46dp"
+                    android:background="@drawable/circle_button_background"
+                    android:onClick="@{() -> vm.onClickArmpit()}">
```

Comment:
For what do we need these wrappers? Could all the attributes be in the `TextView`s maybe? If needed, `FrameLayout` seems a better fit.

Reply:
Attributes moved to the TextView.